Formula/ruby@1.8.7.rb

```
@@ -0,0 +1,205 @@
+      --with-vendordir=#{HOMEBREW_PREFIX}/lib/ruby/vendor_ruby
+    ]
+
+    if build.universal?
+      ENV.universal_binary
+      args << "--with-arch=#{Hardware::CPU.universal_archs.join(",")}"
+    end
+
+    args << "--program-suffix=#{program_suffix}" if build.with? "suffix"
+    args << "--enable-install-doc" if build.with? "doc"
+
+    rm_r "ext/tk" if build.without? "tcltk"
+
+    system "./configure", *args
+
+    # Ruby has been configured to look in the HOMEBREW_PREFIX for the
+    # sitedir and vendordir directories; however we don't actually want to create
+    # them during the install.
+    #
+    # These directories are empty on install; sitedir is used for non-rubygems
+    # third party libraries, and vendordir is used for packager-provided libraries.
+    inreplace "instruby.rb" do |s|
+      s.gsub! "\n    makedirs [archlibdir, sitearchlibdir, vendorarchlibdir]\n",
+              "\n    makedirs [archlibdir]\n"
+      s.gsub! "\n    makedirs [rubylibdir, sitelibdir, vendorlibdir]\n",
+              "\n    makedirs [rubylibdir]\n"
+    end
+
+    system "make"
+    system "make", "install"
+    system "make", "install-doc" if build.with? "doc"
+
+    # Install rubygems into its own prefix, then move into the right place, as
+    # we cannot easily make it install where it should be and the defaults are
+    # no good and would cause a sandbox violation.
+    resource("rubygems").stage do
+      system "#{bin}/ruby#{program_suffix}", "setup.rb", "--prefix=#{prefix}/gem"
+      mv (prefix/"gem/bin").children, bin
+      mv (prefix/"gem/lib").children, lib/"ruby/#{abi_version}"
+      rm_r prefix/"gem"
+    end
+  end
+
+  def post_install
+    # Customize rubygems to look/install in the global gem directory
+    # instead of in the Cellar, making gems last across reinstalls
+    config_file = lib/"ruby/#{abi_version}/rubygems/defaults/operating_system.rb"
+    config_file.unlink if config_file.exist?
```

If that matters `rm_f` ignores any error, but that probably does not matter.